server/athenian/api/openapi/openapi.yaml

```diff
@@ -6830,3 +6882,92 @@ components:
       - developer_id
       - jira_name
       type: object
+    NotifyReleaseRequest:
+      additionalProperties: false
+      description: Request body of `/notify/release`.
+      example:
+      - author: Vadim Markovtsev
+        commit: c3f4d3a
+        name: 0.8.66
+        repository: github.com/athenianco/athenian-api
+        published_at: 2021-02-23T15:17:07.000+01:00
+        url: https://github.com/athenianco/athenian-api/releases/tag/v0.8.66
+      properties:
+        account:
+          description: Account ID.
```

Comment:
It must be resolved by other means, e.g. Auth. We shouldn't allow users to put arbitrary stuff here.

Reply:
It might actually be a good idea to return it in the response, btw.

Reply:
I have to explain how this works.

We allow the same user to be a member of several Athenian accounts at the same time, that's a product-enforced feature. So almost all of our queries require the client to specify their effective account number. The pair `<Account ID> - <User ID>` is checked by searching in `user_accounts` table in the state DB.

Depending on the account number, we put the registered release event to the corresponding account.

Reply:
This is exactly why I think we should limit this to access tokens and avoid use auth for now. It should be bound to a specific account already, so we don't have to deal with authenticated users that belong to more than one org (read: having all state bound already).

This is how GH does it as well: you could have a "deploy keys" for a specific repo.

Imagine that we send a Github install link, start fetching and to speed-up the onboarding we could instantly generate a token that is bound to the account even though it has no `state` and no users. I think it's pretty powerful.

Reply:
OK, so you suggest including the account ID inside the APIKey. Fair enough.

Reply:
Yeah, maybe encode it there or just have a table with `(account, token)` (or whatever works from security PoV).
Also, since releases are bound to the source control, I think it might be useful to bind to `github.accounts` directly. Will require less JOINs to find commits, etc.

Reply:
We already have this table, yes. Regarding the accounts, I'd rather not hardcode them directly because APIKeys are used in other endpoints that would benefit from regular accounts.

Reply:
Agree, for now it doesn't really matter, I think. Later if we decide to optimize it we could query both tables, for example.

Reply:
I see that I thought through APIKeys well last year: the APIKey is an encrypted DB token ID that leads to all the info: user, account, enabled/disabled, permissions, etc.